.circleci/config.yml

```diff
@@ -21,7 +21,7 @@ jobs:
 
   linkchecker:
     docker:
-      - image: yarikoptic/linkchecker:9.4.0.anchorfix1-1
+      - image: markiewicz/linkchecker:v10.0.1-1
```

Comment:
should we may be just establish `bids` or `bids-specification` docker hub org? (otherwise we would just keep footballing it across private orgs)

Reply:
There is a `bids`, but it's maxed out in users. Guess we could create a `bids-maintenance` one for things like this, as we did with Zenodo.

Reply:
> but it's maxed out in users

looks like we can have 3 users for a free-plan dockerhub org. As long as these three are also BIDS maintainers, I think that sufficiently minimizes the bus factor, so I'd rather we check and redistribute the "members" of current `bids` dockerhub than creating a new `bids-maintenance` dockerhub org (which would also be fine, but not preferred from my side)

https://www.docker.com/blog/how-to-manage-docker-hub-organizations-and-teams/

Reply:
`bids` already has 9 users. Should we kick 8 people off to add 2?

Reply:
ah, that was the "grandfathered" status that @rwblair was talking about.

Luckily, out of those 9 people, 3 regularly attend the maintainers meetings (Ross, Franklin, myself). Then there is a certain `bidsappdev` user ... would be nice to get the credentials for that account, because then we could "repurpose" those to be shared among bids maintainers, wdyt?

Reply:
I think creating a shared account between trusted parties is the easiest way forward, and IIRC might avoid some issues etc whenever working with individual accounts. May be just `bidsdev` or just `bids`? (IIRC there is difference between organizations and users on docker hub; so may be should be ok)